draft-birkholz-scitt-receipts.md

```diff
@@ -60,13 +60,55 @@ This document defines a method for countersigning of COSE_Sign1 messages when us
 
 {: #mybody}
 
-# Cryptographic Components
+# Common Parameters
 
-## Merkle Trees
+A Transparency Service backed by a Merkle tree is defined by a collection of immutable parameters, which are used by clients to communicate with the service and to verify artifacts. Each of these parameters must be established before the service operator begins to operate the service. At minimum, the following must be defined:
```

Comment:
communicate = issuing/adding and verifying?

Reply:
That reflects my common struggle on having two separate documents with API not being in here. But yes.